src/Microsoft.OpenApi.OData.Reader/OData/ODataConstants.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.OpenApi.OData
+{
+    internal static class ODataConstants
+    {
+        /// <summary>
+        /// Namespaces used in standard included models.
+        /// </summary>
+        public static IList<string> StandardNamespaces = new List<string>
+        {
+            "Org.OData.",
+            "Edm",
```

Comment:
> "Edm", [](start = 11, length = 7)

others have "." at the end. so how about "Edm."?

Reply:
Th eothers do, but for the main Edm namesapce that is referenced by standard, it never does.  e.g. integer and string are just Edm.integer